gateway/ln-gateway/src/rpc/rpc_server.rs

```diff
@@ -137,7 +139,11 @@ fn v1_routes(gateway: Gateway) -> Router {
     // Public routes on gateway webserver
     let public_routes = Router::new()
         .route("/pay_invoice", post(pay_invoice))
-        .route("/id", get(get_gateway_id));
+        .route("/id", get(get_gateway_id))
+        // These routes are for next generation lightning
+        .route("/payment_info", post(payment_info_ng))
+        .route("/send_payment", post(send_payment_ng))
```

Comment:
This is logically the same operation as `pay_invoice`. Should we use `pay_invoice_v2` to be consistent?